nwdp wq_tools wq_tools.sh is not configured to run on this node
/usr/lib/python3/dist-packages/numpy/core/_methods.py:59: RuntimeWarning: Mean of empty slice.
  warnings.warn("Mean of empty slice.", RuntimeWarning)
/usr/lib/python3/dist-packages/numpy/lib/function_base.py:3558: RuntimeWarning: Invalid value encountered in median
  RuntimeWarning)
110 >70
115 >70
116 >70
117 >70
118 >70
119 >70
120 >70
122 >70
125 >70
127 >70
130 >70
135 >70
110 >45
115 >45
116 >45
117 >45
118 >45
119 >45
120 >45
122 >45
125 >45
127 >45
130 >45
135 >45
110 <35
115 55
116 60
117 70
118 78
119 82
120 >111
122 >111
125 >111
127 >111
130 >111
135 >111
110 <10
115 >60
116 >60
117 >60
118 >60
119 >60
120 >60
122 >60
125 >60
127 >60
130 >60
135 >60
110 16
115 >26
116 >26
117 >26
118 >26
119 >26
120 >26
122 >26
125 >26
127 >26
130 >26
135 >26
110 17
115 >18
116 >18
117 >18
118 >18
119 >18
120 >18
122 >18
125 >18
127 >18
130 >18
135 >18
110 17
115 >17
116 >17
117 >17
118 >17
119 >17
120 >17
122 >17
125 >17
127 >17
130 >17
135 >17
110 5
115 5
116 5
117 5
118 5
119 5
120 5
122 5
125 5
127 5
130 5
135 5
Traceback (most recent call last):
  File "Spill_v_TDG_daily_py36_v1.py", line 228, in <module>
    df1 = get_cwms(paths, public = True, interval = 'hourly', start_date = start, end_date = end, timezone = 'PST')
  File "/usr2/home/rwcds/dx/nwdp/wq_tools/script/cwms_read/cwms_read.py", line 117, in get_cwms
    data = json_data[s]
KeyError: 'WSBW'
